x11-libs/libpciaccess/libpciaccess-0.14.recipe

```diff
@@ -41,13 +51,26 @@ BUILD_PREREQUIRES="
 	cmd:autoconf
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
-	cmd:libtool
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage libpciaccess$secondaryArchSuffix \
+	$libDir/libpciaccess.so.$libVersion
+
+PATCH()
+{
+
```

Comment:
No need for a blank line here.

Oops, that's my fault. Sorry, I had not noticed I had inserted a blank line in my comment.